helpdesk-bot/src/test/resources/jbehave.stories/helpdeskbot_ticket_management.story

```diff
@@ -0,0 +1,74 @@
+Meta: @helpdeskbot
+
+Narrative:
+As a user
+I want to perform a ticket management with bot
+So that I can achieve a business goal
+
+!-- Create ticket scenarios
+
+Scenario: Create a chat (ticket) with helpdeskbot
+Given a new user helpdesk_super_user in a private pod with roles INDIVIDUAL,SUPER_ADMINISTRATOR
+!--And a certificate for API user helpdesk_super_user
+!--When API user helpdesk_super_user authenticates using a certificate
+When user helpdesk_super_user logs in
+And set bot user helpdeskbot
+And user helpdesk_super_user creates an IM with helpdeskbot
+And helpdeskbot create a ticket to help the client helpdesk_super_user
+Then helpdeskbot send a successful message to helpdesk_super_user
+
+Scenario: Try to create new ticket without close other before
+Then helpdesk_super_user talk to helpdeskbot
+And helpdeskbot send a message to ticket room without create new ticket
+And helpdeskbot not send a successful message to helpdesk_super_user
+
+!-- Claim ticket scenarios
+
+Scenario: Claim a ticket
+Given a new user helpdesk_agent in a private pod with roles INDIVIDUAL
+When user helpdesk_agent logs in
+Then user helpdesk_agent join into a queue room
+And user helpdesk_agent claim a ticket in the queue room
+And user helpdesk_agent join into a ticket room
+
+Scenario: Claim a ticket and receive an error
+When user helpdesk_agent logs in
+Then user helpdesk_agent join into a queue room
+And user helpdesk_agent claim a ticket in the queue room
+Then helpdesk_agent receive an error message
+
+!-- Join conversation scenarios
+
+Scenario: Join conversation
+Given a new user helpdesk_agent2 in a private pod with roles INDIVIDUAL
+When user helpdesk_agent2 logs in
+Then user helpdesk_agent2 join into a queue room
+And user helpdesk_agent2 join conversation of a claimed ticket
+And user helpdesk_agent2 join into a ticket room
+
+Scenario: Try to join conversation and receive an error
+When user helpdesk_agent2 logs in
+Then user helpdesk_agent2 join into a queue room
+And user helpdesk_agent2 join conversation of a claimed ticket
+Then helpdesk_agent2 receive an error message
+
+
+!-- Close ticket scenarios
+Scenario: Close ticket
+When helpdesk_super_user send a close message to helpdeskbot
+Then helpdeskbot send a message to helpdesk_super_user
+And helpdeskbot close the ticket room
+
+
+Scenario: Try to close ticket and receive an error
+When helpdesk_super_user send a close message to helpdeskbot
+Then helpdesk_super_user receive an error message
+
+
```

Comment:
- Add scenario to Idle message.
- Add scenario to Claim ticket with show history false on POD, when agent claim ticket, the bot should re-post all messages in service room.
- Other scenario to showHistory false is repeat the last scenario, and after that the agent leave to room, and the bot send idle message, and other agent claim the ticket, all messages with agent1 and user1 is re-posteds.